src/api/dutchx.ts

```diff
@@ -68,6 +68,43 @@ export const getTokenBalances = async (account: Account) => {
   return [ETH, GNO]
 }
 
+// export const lastClosingPrice = async (sellToken: TokenCode, buyToken: TokenCode) => {
+//   const DX = getDutchXConnection()
+//   const exchange = DX[`DutchExchange${sellToken}${buyToken}`]
+//   // const [ account ] = await DX.web3.eth.accounts
+
+//   const currAuctionIdx = (await exchange.auctionIndex()).toNumber()
+//   const [num, den] = await exchange.closingPrices(currAuctionIdx - 1)
+
+//   console.log(`currAuctionIdx = ${currAuctionIdx} // lastClosingPrice: 1 ${sellToken} buys ${num/den} ${buyToken}`)
+
+//   return (num/den)
+// }
+
+/**
+ * closingPrice - get's closingPrice of Auction
+ * @param sellToken 
+ * @param buyToken 
+ * @param aDiff - Number to offset auctionIndex by - if left blank defaults to lastAuction (-1)
+ */
+export const closingPrice = async (sellToken: TokenCode, buyToken: TokenCode, aDiff: number = 1) => {
+  const DX = getDutchXConnection()
+  const exchange = DX[`DutchExchange${sellToken}${buyToken}`]
+  let auctionOffset = aDiff
+
+  const currAuctionIdx = (await exchange.auctionIndex()).toNumber()
+    // Guard against negative index
+  if (currAuctionIdx - aDiff < 0) {
+    console.warn('WARNING: Attempting to access a negative indexed Auction - reverting to current Auction price.')
+    auctionOffset = 0
```

Comment:
no need for `auctionOffset`, just change `aDiff` here directly

Reply:
eslint hates it - why I changed it but yes I agree. I'll relax that rule, thoughts?

Reply:
eslint? tslint
I've been relaxing these rules all day yesterday. Relax away.